commandline/src/main/java/tech/pegasys/web3signer/commandline/config/PicoV3WalletBulkloadParameters.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline.config;
+
+import tech.pegasys.web3signer.commandline.DefaultCommandValues;
+import tech.pegasys.web3signer.signing.config.KeystoresParameters;
+
+import java.nio.file.Path;
+
+import picocli.CommandLine.Option;
+
+public class PicoV3WalletBulkloadParameters implements KeystoresParameters {
+  public static final String WALLETS_PATH = "--wallets-path";
```

Comment:
nit: Wondering if we need to be more specific here or whether this is the only type of wallet likely to be used in this context?

Maybe need to mention v3?

Reply:
- eth1 mode always use v3 keystore (aka wallets) that contains encrypted SECP256K1 private keys
- eth2 mode always use v4 keystore (that we refer as keystore) that contains encrypted BLS private keys
- filecoin mode support(ed) both v3 and v4 keystores.